contrib/init/sysvinit-redhat/docker

```diff
@@ -56,7 +56,7 @@ start() {
         prestart
         printf "Starting $prog:\t"
         echo "\n$(date)\n" >> $logfile
-        "$unshare" -m -- $exec daemon $other_args &>> $logfile &
+        "$unshare" -m -- $exec daemon $other_args & >> $logfile &
```

Comment:
Adding a sapce is not correct, it can't redirect standard output and standard error to the log file. But I don't know why this error will occur in my servers. It looks like depends on differe machine.